src/bundle/Core/EventSubscriber/TrustedHeaderClientIpEventSubscriber.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class TrustedHeaderClientIpEventSubscriber implements EventSubscriberInterface
+{
+    public const PLATFORM_SH_TRUSTED_HEADER_CLIENT_IP = 'X-Client-IP';
```

Comment:
```suggestion
    private const PLATFORM_SH_TRUSTED_HEADER_CLIENT_IP = 'X-Client-IP';
```
unless we want to allow reusing this outside of this class?

Reply:
It is reused in tests: https://github.com/ibexa/core/pull/165/files#diff-fbdb24bdf9edb15e702a2069b2675d9554ddc7ae236cf4a2ca61c24e5f347f5cR83

Reply:
Tests should not rely on class constants, as not using them allows detection when the behavior is changed, so I'd :+1: making it private.